tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
-envlist = py27,py34,pep8
+envlist = py27,py37,pep8
```

Comment:
I believe that it would be good to add matrix for pytest3.x & pytest4.x .
So we have compatibility ensured.